### ISSUE TITLE
Number.prototype.toString() throws warning when called with a radix

### ISSUE THREAD

**Issue by Datamance** [google/closure-compiler#2404]
I should be able to call `Number.prototype.toString` with a radix, [see here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString)

GCC throws an incorrect warning:
```
WARNING - Function num.toString: called with 1 argument(s). Function requires at least 0 argument(s) and at most 0.
            .map(function(num) { return num.toString(32)}).join('')
```

Here's my compiler info:

**VERSION**: 20161201.0.0

**FLAGS**:
java -jar \
  node_modules/google-closure-compiler/compiler.jar \
  --js="nps-plugin/index.js" \
  --js="node_modules/google-closure-library/closure/goog/**.js" \
  --js="!node_modules/google-closure-library/closure/goog/**_test.js" \
  --dependency_mode=STRICT \
  --compilation_level=ADVANCED_OPTIMIZATIONS \
  --entry_point=lnkd.NpsPlugin \
  --output_wrapper="(function(){%output%})();"\
  --new_type_inf \
  --jscomp_warning=newCheckTypes \
  --jscomp_off=newCheckTypesExtraChecks \
  --hide_warnings_for=node_modules/google-closure-library/ \
  --output_manifest="build/report/manifest.MF" \
  --js_output_file=nps-plugin.js

Let me know if you can repro. Thanks!
